Ecierge.Uno.Controls/LocationBreadcrumb/LocationBreadcrumbIterator.cs

```diff
@@ -0,0 +1,101 @@
+#nullable enable
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+#if HAS_UNO
+using Uno.UI.Controls;
+#endif
+
+namespace Ecierge.Uno.Controls.LocationBreadcrumb;
+
```

Comment:
Этот класс используется?
```suggestion
#nullable enable
namespace Ecierge.Uno.Controls;

using System;
using System.Collections;
using System.Collections.Generic;
#if HAS_UNO
using Uno.UI.Controls;
#endif

```